crates/astria-cli/src/commands/sequencer.rs

```diff
@@ -111,7 +107,7 @@ pub(crate) async fn get_balance(args: &BasicAccountArgs) -> eyre::Result<()> {
 
     println!("Balances for address: {}", args.address);
     for balance in res.balances {
-        println!("    asset ID: {}", balance.denom.id());
+        println!("    asset ID: {}", balance.denom.to_ibc_prefixed());
```

Comment:
this is somewhat confusing as it'll print the native asset as IBC-prefixed when it isn't an IBC'd asset - maybe just leave as the 32-byte ID?

Reply:
IMO we should be consistent everywhere when communicating assets to users: as long as the string that you put into an action as an asset adheres to our ibc ics20 spec, then it will be accepted by sequencer.

Similarly, CLI tools should also return a valid denom so that you can just take it and paste it without having to know that you need to prefix it with `ibc/`.

What do you think if I change the line to read `asset IBC form:`?

Reply:
Following this line of thought, I have decided to just remove it completely for now: sequencer right now returns the `ibc/<sha256>` right now, so the CLI tool doesn't need to print it twice. We can follow up retouching the tool to return both ibc and full trace-prefixed. ac4fe3